test/rules/src/https_everywhere_checker/http_client.py

```diff
@@ -46,8 +46,9 @@ def getCAPath(self, platform):
 class FetchOptions(object):
     """HTTP fetcher options like timeouts."""
 
-    # The default list of cipher suites that ships with Firefox 51.0.1
-    _DEFAULT_CIPHERLIST = "ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES256-SHA:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES128-SHA:ECDHE-RSA-AES256-SHA:DHE-RSA-AES128-SHA:DHE-RSA-AES256-SHA:AES128-SHA:AES256-SHA:DES-CBC3-SHA"
+    # The default list of cipher suites that ships with Firefox 68.5.0 ESR
+    # TLS 1.2 Recommemdation: https://wiki.mozilla.org/Security/Server_Side_TLS
+    _DEFAULT_CIPHERLIST = "ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384"
```

Comment:
`DHE-RSA-AES128-GCM-SHA256` and `DHE-RSA-AES256-GCM-SHA384` are only supported by Firefox and not Chrome. I think it should not be included here.

Also to avoid confusion, the list at https://wiki.mozilla.org/Security/Server_Side_TLS is intended to be used as a HTTPS server configuration, not for use in a browser.

I suggest keeping only:

```
ECDHE-ECDSA-AES128-GCM-SHA256
ECDHE-RSA-AES128-GCM-SHA256
ECDHE-ECDSA-CHACHA20-POLY1305
ECDHE-RSA-CHACHA20-POLY1305
ECDHE-ECDSA-AES256-GCM-SHA384
ECDHE-RSA-AES256-GCM-SHA384
```

(these are supported by both Firefox and Chrome, and are all considered fully secure at the moment)

Reply:
Cross checking this list with https://clienttest.ssllabs.com and https://wiki.mozilla.org/Security/Cipher_Suites, I also agree to keep these as well, the other listed weak ciphers I will double check. But for now this list seems like the cross supported suites.

Reply:
For the future, I will add a comment linking into the wiki on how to update this cipher list

Reply:
@zoracon I think the list should only include the ciphers supported by all three supported browsers (Chrome, Firefox and Tor), since only websites that successfully load over HTTPS on all three should be rewritten to avoid issues.

Reply:
I also cross checked between Tor for this list as well. It looks like it checks out for all 3 browsers. Pushing this change soon.